apiserver/Dockerfile

```
@@ -17,9 +17,10 @@ ARG CALICO_BASE
 FROM scratch AS source
 
 ARG BIN_DIR
+ARG LICENSE_DIR
 ARG TARGETARCH
 
-COPY ${BIN_DIR}/LICENSE /licenses/LICENSE
+COPY ${LICENSE_DIR}/LICENSE /licenses/LICENSE
```

We need an extra argument for this since it is always the same value. Let's just keep things simple.

```suggestion
COPY bin/LICENSE /licenses/LICENSE
```